.github/workflows/rust.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Build
       run: cargo build --verbose --examples
     - name: Run tests
-      run: SCYLLA_URI=172.42.0.2:9042 SCYLLA_URI2=172.42.0.3:9042 SCYLLA_URI3=172.42.0.4:9042 cargo test --verbose
+      run: SCYLLA_URI=172.42.0.2:9042 SCYLLA_URI2=172.42.0.3:9042 SCYLLA_URI3=172.42.0.4:9042 cargo test --verbose --package scylla
```

Comment:
AFAIU, this means that we no longer run tests for other crates (`scylla-cql`, `scylla-proxy`, `scylla-macros`) in the CI. This probably isn't the desired state.

Reply:
Enabled tests for other crates in `rust` and `cassandra` workflows.